bin/lib/releases.py

```diff
@@ -13,7 +13,7 @@ def __str__(self) -> str:
 
 
 class VersionSource(Enum):
-    value: Tuple[int, str]
+    value: Tuple[int, str]  # type: ignore[assignment]
```

Comment:
I missed this in my review - what is this all about?

Reply:
https://docs.python.org/3/whatsnew/3.11.html#enum causes mypy to fail.